conans/client/cmd/export.py

```diff
@@ -32,13 +32,25 @@ def cmd_export(conanfile_path, name, version, user, channel, keep_source,
     conan_linter(conanfile_path, output)
     conanfile = _load_export_conanfile(conanfile_path, output, name, version)
     conan_ref = ConanFileReference(conanfile.name, conanfile.version, user, channel)
+    if conan_ref.revision:  # Could be improved modifying the recipe?
```

Comment:
Now I'm not allowing to export specifying a revision in the command line, it could be a later improvement and will require changing the conanfile. 
Also, a flag (--latest-revision=remote) that could check the latest revision in a remote and export with +1 is not implemented either.